os/mm/umm_heap/umm_mallinfo.c

```diff
@@ -87,25 +87,33 @@
 
 struct mallinfo mallinfo(void)
 {
-	int heap_idx;
 	struct mallinfo info;
 #if CONFIG_KMM_NHEAPS > 1
 	memset(&info, 0, sizeof(struct mallinfo));
 #endif
+#ifdef CONFIG_APP_BINARY_SEPARATION
+	mm_mallinfo(&BASE_HEAP[0], &info);
```

Comment:
I think BASE_HEAP is enough

Reply:
For easy understanding, I will keep this. Thanks.